src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -461,27 +463,23 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
     return disabledChains.filter((chainId: number) => !isNaN(chainId) && Number.isInteger(chainId) && chainId !== 1);
   }
 
-  async getUBATargetSpokeBalances(
-    chainIds: number[],
-    l1TokenAddress: string,
-    blockNumber?: number
-  ): Promise<{ spokeChainId: number; target: BigNumber }[]> {
-    return chainIds.map((chainId) => {
-      const target = this.getSpokeTargetBalancesForBlock(l1TokenAddress, chainId, blockNumber).target;
-      return { spokeChainId: chainId, target };
-    });
-  }
-
   /**
-   * Retrieves a UBA config for a given L1 token address and block number.
+   * Retrieves the most recently set UBA config for a given L1 token address before a block number.
    * @param l1TokenAddress The L1 token address to retrieve the config for
-   * @param blockNumber The block number to retrieve the config for. If not provided, the latest config will be returned.
-   * @returns The UBA config for the given L1 token address and block number, or undefined if no config exists.
+   * @param blockNumber The block number to retrieve the config for. If not specified, sets block to max integer
+   * meaning that this function will return the latest config.
+   * @returns The UBA config for the given L1 token address and block number, or undefined if no config exists
+   * before blockNumber.
    */
-  getUBAConfig(l1TokenAddress: string, blockNumber?: number): UBAParsedConfigType | undefined {
+  getUBAConfig(l1TokenAddress: string, blockNumber = Number.MAX_SAFE_INTEGER): UBAParsedConfigType | undefined {
```

Comment:
Nit should we make this a public function and explicitly set the access modifier?
```suggestion
  public getUBAConfig(l1TokenAddress: string, blockNumber = Number.MAX_SAFE_INTEGER): UBAParsedConfigType | undefined {
```